pkg/volume/csi/csi_mounter.go

```diff
@@ -392,12 +392,15 @@ func (c *csiMountMgr) supportsFSGroup(fsType string, fsGroup *int64, driverPolic
 		return false
 	}
 
-	accessModes := c.spec.PersistentVolume.Spec.AccessModes
+	if c.spec.PersistentVolume == nil {
+		klog.V(4).Info(log("mounter.SetupAt Warning: skipping fsGroup permission change, no access mode available. The volume may only be accessible to root users."))
+		return false
```

Comment:
Is there really no way to support fsGroup for CSI ephemeral inline volumes?

Reply:
there is a way. I left a comment above https://github.com/kubernetes/kubernetes/pull/102653#discussion_r646889247 explaining that.